src/migrations/20210428103923-onboard-projects-to-rbac.js

```diff
@@ -0,0 +1,63 @@
+const async = require('async');
+
+const DESCRIPTION = {
+    OWNER:
+        'Users with this role have full control over the project, and can add and manage other users within the project context, manage feature toggles within the project, and control advanced project features like archiving and deleting the project.',
+    MEMBER:
+        'Users with this role within a project are allowed to view, create and update feature toggles, but have limited permissions in regards to managing the projects user access and can not archive or delete the project.',
+};
+exports.up = function(db, cb) {
+    db.runSql(
+        `SELECT id AS name from projects WHERE id NOT IN (SELECT DISTINCT project FROM roles WHERE project IS NOT null)`,
+        (err, results) => {
+            if (results && results.rowCount > 0) {
+                const projects = results.rows;
+                const createProjectRoles = projects.map(p =>
+                    db.runSql.bind(
+                        db,
+                        `
+                WITH project_owner AS (
+                    INSERT into roles (name, description, type, project)
+                        VALUES ('Owner', '${DESCRIPTION.OWNER}', 'project', '${p.name}')
+                        RETURNING id role_id
+                )
+                INSERT INTO role_permission(role_id, project, permission)
+                SELECT role_id, '${p.name}', 'ADMIN' from project_owner;
+
+                WITH project_member AS (
+                    INSERT into roles (name, description, type, project)
+                        VALUES ('Member', '${DESCRIPTION.MEMBER}', 'project', '${p.name}')
+                        RETURNING id role_id
+                )
+                INSERT INTO role_permission(role_id, project, permission) VALUES
+                    ((SELECT role_id from project_member), '${p.name}', 'CREATE_STRATEGY'),
+                    ((SELECT role_id from project_member), '${p.name}', 'UPDATE_STRATEGY'),
+                    ((SELECT role_id from project_member), '${p.name}', 'DELETE_STRATEGY'),
+                    ((SELECT role_id from project_member), '${p.name}', 'UPDATE_APPLICATION'),
+                    ((SELECT role_id from project_member), '${p.name}', 'CREATE_CONTEXT_FIELD'),
+                    ((SELECT role_id from project_member), '${p.name}', 'UPDATE_CONTEXT_FIELD'),
+                    ((SELECT role_id from project_member), '${p.name}', 'DELETE_CONTEXT_FIELD'),
```

Comment:
These are actually not project permissions. Only feature toggles lives under a project.

Reply:
Good point. I'll fix :)

Reply:
fixed in caa7225352fcd4469a138fa05575271e35aa8686